tools/debian/rules

```diff
@@ -43,19 +43,27 @@ override_dh_auto_configure: debian/git-patches-applied
 		--with-pamdir=/lib/$(DEB_HOST_MULTIARCH)/security \
 		--libexecdir=/usr/lib/cockpit $(CONFIG_OPTIONS)
 
-override_dh_auto_install:
-	dh_auto_install
-	mkdir -p debian/tmp/etc/pam.d
-	install -p -m 644 tools/cockpit.debian.pam debian/tmp/etc/pam.d/cockpit
-
 override_dh_systemd_enable:
 	dh_systemd_enable -p cockpit-ws --name=cockpit cockpit.socket
 
 override_dh_systemd_start:
 	dh_systemd_start -p cockpit-ws cockpit.socket
 
 override_dh_install:
-	dh_install --list-missing -Xusr/src/debug
+	# Debian/Ubuntu PAM config
+	mkdir -p debian/tmp/etc/pam.d
+	install -p -m 644 tools/cockpit.debian.pam debian/tmp/etc/pam.d/cockpit
+
+	# don't ship broken branding symlinks
+	for d in rhel fedora centos; do rm -r debian/tmp/usr/share/cockpit/branding/$$d; done
+	dpkg-vendor --derives-from ubuntu || rm -r debian/tmp/usr/share/cockpit/branding/ubuntu
+
+	# unpackaged modules
```

Comment:
Also note that RPMs behave like `--fail-missing` by default. Our spec file already has plenty of rm commands for the exact same purpose.

For symmetry I adjusted the commit to also remove the broken debian/ubuntu branding symlinks from the RPMs. It's fairly safe to say that in the forseeable future Debian won't move to RPM ☺.

Untagging "bot" as this should now run tests on all OSes.

I also fixed the commit message typo.